plane/source/docs/quadplane-auto-mode.rst

```diff
@@ -55,6 +55,15 @@ will be done at whatever height the aircraft is at when it starts on
 the NAV_VTOL_LAND waypoint. So you would typically want the previous
 waypoint to have an altitude of about 20 meters above the ground.
 
+If Q_OPTIONS is set for "Use a fixed wing approach" (bit 4), then instead of transitioning to VTOL flight and doing a VTOL landing, it will remain in plane mode , and proceed to the landing position,climbing or descending to the altitude set in the NAV_VTOL_LAND waypoint. When it reaches within Q_FW_LND_APR_RAD of the landing location, it will perform a LOITER_TO_ALT to finish the climb or decent to that altitude set in the waypoint, then, turning into the wind, transition to VTOL mode and proceed to the landing location and land.
```

Comment:
`landing position,climbing or descending` should be `landing position, climbing or descending` (missing whitespace after a comma)

`descent` rather then `decent`